src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenByAuthorizationCodeParameterBuilder.cs

```diff
@@ -88,5 +88,18 @@ public AcquireTokenByAuthorizationCodeParameterBuilder WithSendX5C(bool withSend
             Parameters.SendX5C = withSendX5C;
             return this;
         }
+
+        /// <summary>
+        /// Used to secure authorization code grant via Proof of Key for Code Exchange (PKCE).
+        /// See (https://tools.ietf.org/html/rfc7636) for more details.
+        /// </summary>
+        /// <param name="PkceCodeVerifier">A dynamically created cryptographically random key used to provide proof of possession for the authorization code.
+        /// </param>
+        /// <returns>The builder to chain the .With methods</returns>
+        public AcquireTokenByAuthorizationCodeParameterBuilder WithCodeVerifier(string PkceCodeVerifier)
```

Comment:
The naming is inconsistent, on `GetAuthorizationRequestUrl` we have: 

```
authUriBuilder.WithPkce(out codeVerifier);
```

and on `AcquireTokenByAuthorizationCode` we have: 

```
.WithCodeVerifier(codeVerifier)
```

@jmprieur - thoughts?

Reply:
yes, i noticed that yesterday, but thought we should discuss here and not the other PR.

Reply:
So what do we think? Should both of the methods be "WithPkce" ?

Reply:
What about `WithPkceCodeVerifier`?